src/sonic-pit/pit-sysdiag/src/pit_util_common.py

```diff
@@ -0,0 +1,364 @@
+# -*- coding:utf-8 -*-
+from __future__ import print_function
+#from subprocess import PIPE, Popen
+from errcode import E
+import subprocess
+import threading
+import time
+import sys
+import signal
+import pexpect
+import imp
+import re
+import json
+
+# from threading import Lock, Thread
+PLATFORM_ROOT_PATH = "/usr/share/sonic/device"
+PLATFORM_ROOT_PATH_DOCKER = "/usr/share/sonic/platform"
+
+SONIC_CFGGEN_PATH = "/usr/local/bin/sonic-cfggen"
+HWSKU_KEY = "DEVICE_METADATA['localhost']['hwsku']"
+PLATFORM_KEY = "DEVICE_METADATA['localhost']['platform']"
+
+
+def get_platform_and_hwsku():
```

Comment:
several functions in this file are duplicated with the file "function.py", please consider to only keep one of them.

Reply:
Replaced with sonic_py_common.device_info.get_platform_and_hw_sku()